src/Controls/src/Core/Tweener.cs

```diff
@@ -69,119 +81,127 @@ public Tweener(uint length, uint rate, IAnimationManager animationManager)
 			Value = 0.0f;
 			Length = length;
 			Rate = rate;
-			this.animationManager = animationManager;
+			_animationManager = animationManager;
 			Loop = false;
 		}
 
 		public AnimatableKey Handle { get; set; }
 
 		public uint Length { get; }
 
-		public uint Rate { get; }
+		public uint Rate { get; } = 1;
```

Comment:
Why is this needed? It is set in the ctor.

Reply:
Okay, removed from constructor.